src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -215,6 +214,7 @@
     <Compile Include="System\Text\Json\Node\JsonNode.cs" />
     <Compile Include="System\Text\Json\Node\JsonNumber.cs" />
     <Compile Include="System\Text\Json\Node\JsonObject.cs" />
+    <Compile Include="System\Text\Json\Node\JsonObjectEnumerator.cs" />
```

Comment:
Can you also clean-up the bottom of this .csproj -- it has several files added in a separate ItemGroup that should just be merged in with the main ItemGroup.

e.g.
`<Compile Include="System\Text\Json\Node\DuplicatePropertyNameHandling.cs" />`

Reply:
I actually move this one to the bottom as well, we decided to keep them seperatly: https://github.com/dotnet/corefx/pull/40107#discussion_r311761318